msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/enums/EMonsterType.java

```diff
@@ -0,0 +1,22 @@
+package cz.muni.fi.pa165.monsterslayeragency.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * @author Michaela Bajanova (469166)
+ */
+@Getter
+@AllArgsConstructor
+public enum EMonsterType {
+    DRAGON(EFood.MEAT, new EPower[] {EPower.FIRE, EPower.LIGHTNING}, new EPower[] {EPower.ICE}),
```

Comment:
We can make it bit simpler this way but I would say all of these are basicly monsters not a monster type and I kinda liked more when the strenhts and weaknesses were in class monster as every monster has its own.